api/pacman-api-asset/src/main/java/com/tmobile/pacman/api/asset/controller/CloudNotificationsController.java

```diff
@@ -74,14 +75,17 @@ public ResponseEntity<Object> getlistOfCloudNotifications(@RequestBody(required
 			Date endDate=null;
 
 			String searchText = request.getSearchtext();
-			Map<String, String> filter = request.getFilter();
+			Map<String,List<String>> filter = request.getFilter();
 			Map<String,Object> sortFilter=request.getSortFilter();
 
 			if(filter.get("_loaddate")!=null) {
-				String dateRange = filter.get("_loaddate");
-				String[] dates = dateRange.split(" - ");
-				SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
-				dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+				List<String> dateRangeList = filter.get("_loaddate");
+				StringBuilder dateRange= new StringBuilder();
+				for(String range:dateRangeList){
+					dateRange.append(range);
+				}
+				String[] dates = dateRange.toString().split(" - ");
+				SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");

```

Comment:
if it can be logically named (based on its usage, for example) then yes, let's have it as a constant. Otherwise I'm fine with it being like this because date format is very specific and my  be applicable to only this situation.

Reply:
done